modules/OCPPConfiguration/main/mapping_reader.hpp

```diff
@@ -0,0 +1,37 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Pionix GmbH and Contributors to EVerest
+
+#pragma once
+
+#include "everest_config_mapping.hpp"
+#include "generated/types/ocpp.hpp"
+#include <c4/yml/std/map.hpp>
+#include <filesystem>
+#include <ryml.hpp>
+#include <string>
+#include <unordered_map>
+#include <vector>
+
+namespace module {
+
+using types::ocpp::Component;
+using types::ocpp::ComponentVariable;
+using types::ocpp::Variable;
+
+using OcppToEverestConfigMapping = std::unordered_map<ComponentVariable, EverestConfigMapping>;
+
+class MappingReader {
```

Comment:
Please add a brief description for this class

Reply:
Done